spec/jobs/shared_examples/enqueues_job.rb

```diff
@@ -1,7 +1,6 @@
 RSpec.shared_examples "#enqueues_job" do |queue_name, args|
   describe "#perform_later" do
     it "enqueues the job" do
-      ActiveJob::Base.queue_adapter = :test
```

Comment:
This, and the other statements like this, is not needed anymore because we globally set the `queue_adapter` to `:test`